classes/file/TemporaryFileManager.inc.php

```diff
@@ -57,6 +57,16 @@ function deleteFile($fileId, $userId) {
 	}
 
 	/**
+	 * Delete a file by path.
+	 * For temporary files that are not in the DB, e.g. exported xml files.
+	 * @param $filePath string the location of the file to be deleted
+	 * @return boolean returns true if successful
+	 */
+	function deleteFileByPath($filePath) {
```

Comment:
Yes, I wasn't sure -- it seemed to me to be to much 'action' to write and read from the database and act like this would be like a file upload (considering those information as at a file upload), because this case is a little bit different -- those exported files are not like file uploads, that are usually treated by this manager -- they just write a content to a file. Do you think it would be better to do so?